python/rikai/types/vision.py

```diff
@@ -40,12 +40,20 @@ def __init__(self, uri: str):
     def __repr__(self) -> str:
         return f"Image(uri={self.uri})"
 
-    def _repr_html_(self):
-        """Jupyter integration
+    def _repr_mimebundle_(self, include=None, exclude=None):
+        from IPython.display import Image
 
-        TODO: find more appropriate way to return image in jupyter?
-        """
-        return f"<img src={self.uri} />"
+        return Image(self.uri)._repr_mimebundle_(include=include, exclude=exclude)
+
+    def _repr_jpeg_(self):
+        from IPython.display import Image
+
+        return Image(self.uri)._repr_jpeg_()
```

Comment:
Does this work with `s3://` `gs://` URIs?

Reply:
let me test that (at least with s3)